.releaserc.json

```diff
@@ -0,0 +1,113 @@
+{
+  "branches": [
+    {
+      "name": "main"
+    }
+  ],
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "releaseRules": [
```

Comment:
Is there any reason we don't use the convential commit defaults?

Reply:
I have reused these rules from the [shogun-gis-client project](https://github.com/terrestris/shogun-gis-client/blob/main/.releaserc.json). I thought to keep the rules consistent, we use the same rules for all of our projects.

For commitlint, the `@commitlint/config-conventional` are used and also extended by the configuration used by [shogun-gis-client](https://github.com/terrestris/shogun-gis-client/blob/main/.commitlintrc.json).

Reply:
Ah, good catch! Than I would favor to fix this in the gis-client as well ;-)

Reply:
E.g. like [here](https://github.com/terrestris/shogun/blob/main/.releaserc.json).

Reply:
All right, I have now adapted the `releaserc.json`. Thanks!